lib/tasks/relink_ico_to_foi_ir.rake

```diff
@@ -22,15 +22,19 @@ namespace :ico_appeal do
         puts ico_appeal_foi_ids_map
       end
       counter = 0
-      ico_appeal_cases = Case::ICO::Base.all
+      ico_appeal_cases = Case::ICO::FOI.all
       ico_appeal_cases.each do | ico_appeal |
         puts "Checking ico appeal #{ico_appeal.number}"
         foi_ir_id = find_related_foi_ir_case_id(ico_appeal, ico_appeal_foi_ids_map)
         if foi_ir_id.present?
           foi_ir_case = Case::Base.find(foi_ir_id)
-          clear_up_old_links(ico_appeal, 'related')
+          existing_original_case_id = ico_appeal.original_case.id
+          clear_up_old_related_link(ico_appeal.id, foi_ir_id)
```

Comment:
Should this be plural? `clear_up_old_related_links` or is it only one link now? :)

Reply:
@willmcb, related links can be more than one, but for this occasion, I clear up specific related link based on the linked_case_id. so it is only one link